joomla/joomla-cms
# [4.0] Frontend: correcting Tags layouts

### Summary of Changes
Following up on similar layouts for Articles List and Contact lists, this one concerns the 3 Tags layouts.
Getting rid of fieldset, separating search filter from list limit. some a11y.
Deleting one redundant lang string, modifying another.
Adding a missing parameter field in xml.

Kept one layout as table, left the others as ul/li.

### Testing Instructions
Create some tags. Tag some contacts and articles.

Create 3 menu items
1. Tagged Items
2. List all tags
3. Compact List of Tagged Items

Display in frontend
patch and test again.

Check code produced
### Actual result BEFORE applying this Pull Request
1. Tagged Items
<img width="873" alt="Screen Shot 2020-10-08 at 17 34 37" src="https://user-images.githubusercontent.com/869724/95480776-954f3680-098c-11eb-9805-be042a8d36c3.png">

2. List all tags
<img width="869" alt="Screen Shot 2020-10-08 at 17 35 03" src="https://user-images.githubusercontent.com/869724/95480857-aa2bca00-098c-11eb-9c8a-b3f0cd6e7613.png">

3. Compact List of Tagged Items
<img width="869" alt="Screen Shot 2020-10-08 at 17 35 40" src="https://user-images.githubusercontent.com/869724/95480907-b879e600-098c-11eb-9e0f-eecb0f988e79.png">



### Expected result AFTER applying this Pull Request
1. Tagged Items
<img width="897" alt="Screen Shot 2020-10-08 at 17 31 34" src="https://user-images.githubusercontent.com/869724/95480489-40abbb80-098c-11eb-8194-85d03a3c8f89.png">

2. List all tags
<img width="872" alt="Screen Shot 2020-10-08 at 17 32 47" src="https://user-images.githubusercontent.com/869724/95480562-502b0480-098c-11eb-89a2-acf18e15a358.png">

3. Compact List of Tagged Items
<img width="891" alt="Screen Shot 2020-10-08 at 17 33 13" src="https://user-images.githubusercontent.com/869724/95480625-60db7a80-098c-11eb-811b-a0ebd51b463f.png">
